src/App.js

```diff
@@ -13,14 +18,39 @@ function App() {
   // the "5" button, or the operator if they click one of those buttons) and then call your setter function to update state.
   // Don't forget to pass the functions (and any additional data needed) to the components as props
 
-  return (
-    <div className="container">
-      <Logo />
-      <div className="App">
-        {/* STEP 4 - Render your components here and be sure to properly import/export all files */}
-      </div>
-    </div>
-  );
+ 
+  const [currentNum, setCurrentNum] = useState(" ");
+  // const [operator, setOperator] = useState();
+  
+  const addNumber = (number) => {
+    setCurrentNum(currentNum => currentNum + number);
+  };
+  const addOperator = (operator) => { 
+  if (operator === "=") {
+    setCurrentNum(currentNum => eval(currentNum)) 
+  } else {
+    setCurrentNum(currentNum => currentNum + " " + operator + " ");
+  }
+   
+  };
+
+	return (
+		<div className="container">
```

Comment:
total design choice: I personally prefer code with just 2 spaces instead of 4, it's a little easier to read. There should be a setting inside prettier